src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -176,6 +176,11 @@ def fs_status(self, fs_id):
                     activity = 0.0
 
                 metadata = mgr.get_metadata('mds', info['name'])
+                if metadata is None:
+                    raise DashboardException(
```

Comment:
If we're raising an exception in this case, we need to update the dashboard cephfs QA tests, because they are expecting this call to always succeed.

Also, if we get a 500 error, what should be the behavior of the test? Should it retry the request?